### PR TITLE
Added Discord Invite Link

### LINKED ISSUE #4: Modifying owner-name and repository-name to be dynamic in README
## Description

It would be cool if we can make the README instructions to use the action to have a dynamic name.

It was a little bit confusing for me to see that it is referring to the same repository that we are in.
I think that changing the **owner name** and the **repository name** to be dynamic will be helpful.

## Screenshot
Screen Shot 2021-04-01 at 19 43 55

## Before Changing

```yaml
         owner-name: 'EddieHubCommunity'
         repository-name: 'gh-action-open-source-labels'
```

## After Changing

```yaml
          owner-name: ${{ github.repository_owner }}
          repository-name: ${{ github.event.repository.name }}
```

### LINKED ISSUE #5: Add a discord link to the README
Add a Discord Link for the people who haven't joined the open-source community and also the heading of the README says Community Template repo so how about it ?

### PR DESCRIPTION
**Pull request by Prathamesh-Shanbhag**
Fixes  #4 & #5 

## Proposed Changes

  - Added Discord Invite Link to Readme
  - Changed owner and repo name to be dynamic
  -

## Screenshots
Screenshot 2021-04-02 173240

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ jobs:
     - uses: EddieHubCommunity/gh-action-open-source-labels@main
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
-        owner-name: 'EddieHubCommunity'
-        repository-name: 'gh-action-open-source-labels'
+       owner-name: ${{ github.repository_owner }}
+       repository-name: ${{ github.event.repository.name }}
```
Comment (Rahma-AlWadhahi) [inline README.md:29]: Could you add `one space` to make it look consistent and avoid `yaml` file errors!?

```suggestion
        owner-name: ${{ github.repository_owner }}
        repository-name: ${{ github.event.repository.name }}
```
#### Before changing
```yaml
    - uses: EddieHubCommunity/gh-action-open-source-labels@main
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
       owner-name: ${{ github.repository_owner }}
       repository-name: ${{ github.event.repository.name }}
```

#### After changing
```yaml
    - uses: EddieHubCommunity/gh-action-open-source-labels@main
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
        owner-name: ${{ github.repository_owner }}
        repository-name: ${{ github.event.repository.name }}
```

Comment (Prathamesh-Shanbhag) [reply README.md:29]: Yes I will make the changes!

**Review by schmelto — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by eddiejaoude** [comment]
Great collaboration, thank you all 💥

**Pull request merged**